src/pages/docs/messages/annotations.mdx

```diff
@@ -0,0 +1,529 @@
+---
+title: Message Annotations
+meta_description: "Annotate messages on a channel with additional metadata."
+languages:
+  - javascript
+  - nodejs
+---
+
+<Aside data-type='experimental'>
+Message Annotations is currently Experimental. Its features are still in development and subject to rapid change.
+</Aside>
+
+Message annotations allow clients to add metadata to existing messages on a channel. You can use annotations to implement features like:
+
+- **Message reactions** - Add emoji reactions (👍, ❤️, 😂) to messages
+- **Content categorization** - Tag messages with categories like "important" or "urgent"
+- **Community moderation** - Flag inappropriate content for review
+- **Read receipts** - Mark messages as "read" or "delivered"
+
+When clients publish or delete annotations, Ably automatically creates a [summary](#annotation-summaries) that provides an aggregated view of all annotations for a message.
+
+
+## Enable annotations <a id="enable"/>
+
+Annotations can be enabled for a channel or channel namespace with the *Advanced message features* channel rule.
+
+<Aside data-type='important'>
+Note that when advanced messaging is enabled, messages are [persisted](/docs/storage-history/storage#all-message-persistence) by default, and [continuous history](/docs/storage-history/history#continuous-history) features are not currently supported.
+</Aside>
+
+1. On your [dashboard](https://ably.com/accounts/any), select one of your apps.
+2. Go to **Settings**.
+3. Under [channel rules](/docs/channels#rules), click **Add new rule**.
+4. Enter the channel name or channel namespace on which to enable message annotations.
+5. Check **Advanced message features** to enable message annotations.
+6. Click **Create channel rule** to save.
+
+
+## Annotation properties <a id="annotation-properties"/>
```

Comment:
I'd suggest we move this below publish / subscribe as it's not really the first thing you need to know, or understand.